codespell_lib/_shtab.py

```diff
@@ -0,0 +1,8 @@
+FILE = None
+DIRECTORY = DIR = None
+
+
+def add_argument_to(parser, *args, **kwargs):
+    from argparse import Action
+    Action.complete = None
+    return parser
```

Comment:
From what I can see this file is basically a NOP/DFA stub for when shtab isn't present, could we have a comment to that affect in it, or a pydoc or whatever?

Reply:
I'll fix it.